tests/integration_tests.py

```diff
@@ -647,6 +647,90 @@ def test_will_not_output_empty_state(self):
                           {"bookmarks":{"chicken_stream":{"id": 1 }},
                            'currently_syncing' : 'chicken_stream'})
 
+class SingleRecordBatches(unittest.TestCase):
+    def setUp(self):
+        self.maxDiff = None
+        token = None
+        handler = StitchHandler(target_stitch.DEFAULT_MAX_BATCH_BYTES, 3)
+
+        self.og_check_send_exception = target_stitch.check_send_exception
+        self.out = io.StringIO()
+        self.target_stitch = target_stitch.TargetStitch(
+            [handler], self.out, 4000000, 2, 100000)
+        self.queue = [json.dumps({"type": "SCHEMA", "stream": "chicken_stream",
+                                  "key_properties": ["id"],
+                                  "schema": {"type": "object",
+                                             "properties": {"id": {"type": "integer"},
+                                                            "name": {"type": "string"}}}}),
+                      json.dumps({"type": "SCHEMA", "stream": "zebra_stream",
+                                  "key_properties": ["id"],
+                                  "schema": {"type": "object",
+                                             "properties": {"id": {"type": "integer"},
+                                                            "name": {"type": "string"}}}})]
+
+        target_stitch.SEND_EXCEPTION = None
+        for f,s in target_stitch.PENDING_REQUESTS:
+            try:
+                f.cancel()
+            except:
+                pass
+
+        target_stitch.PENDING_REQUESTS = []
+        LOGGER.info("cleaning SEND_EXCEPTIONS: %s AND PENDING_REQUESTS: %s",
+                    target_stitch.SEND_EXCEPTION,
+                    target_stitch.PENDING_REQUESTS)
+
+        target_stitch.CONFIG ={
+            'token': "some-token",
+            'client_id': "some-client",
+            'disable_collection': True,
+            'connection_ns': "some-ns",
+            'batch_size_preferences' : {
+                'full_table_streams' : [],
+                'batch_size_preference': None,
+                'user_batch_size_preference': None,
+            },
+            'turbo_boost_factor' : 10,
+            'small_batch_url' : "http://small-batch",
+            'big_batch_url' : "http://big-batch",
+        }
+
+    def test_single_record_batches_dont_exist(self):
+        # Tests that the target will not emit single record batches if
+        # streams alternate rapidly due to this increasing the chance for
+        # duplicate sequence numbers.
```

Comment:
This kind of language should probly be removed. It's not about sequence numbers anymore. The other PR handled tests for sequence numbers.

Reply:
Updated the language in the test to more accurately reflect the intent.